facefusion/core.py

```diff
@@ -34,8 +34,15 @@
 
 def cli() -> None:
 	signal.signal(signal.SIGINT, lambda signal_number, frame: destroy())
-	program = ArgumentParser(formatter_class = lambda prog: HelpFormatter(prog, max_help_position = 160), add_help = False)
-	# general
+	program = ArgumentParser(formatter_class = lambda prog: HelpFormatter(prog, max_help_position=160), add_help = False)
+    # general
+	program.add_argument('-c', '--config_path', help = wording.get('help.config_path'), dest = 'config_path', default = 'facefusion.ini')
+	args = program.parse_args()
```

Comment:
there should be any set of globals or validation in this method...

1. move the pre parse to the very top of apply_args()
2. move the config file validation to the config.py
3. do not hardcode wording
4. no need for facefusion.globals.config_path = 'facefusion.ini' when it is the default

Reply:
For number 4. I'm pretty sure that was so that if you don't run any arguments it uses facefusion.ini. Because we simplified the get_config() to just use the global but the global is empty so it wouldn't actually use facefusion.ini

Reply:
okay, it might no longer be needed once facefusion.ini is a argument default